scripts/Email-Sender/script.py

```diff
@@ -0,0 +1,21 @@
+# Script to automate Email Sending
+import smtplib
+
+to = input("Enter the Email of recipent:\n")
+content = input("Enter the Content for E-Mail:\n")
+
+sender_email = "youremail@mail.com"
+sender_password = "yourpassword"
```

Comment:
Could you please add an input field for the user to provide this during runtime?

Reply:
Sure